Makefile

```diff
@@ -16,6 +16,7 @@ format: FORCE
 
 test: lint FORCE
 	pytest -v test
+	FUNSOR_GENERIC_SUBS=1 pytest -v test  # TODO remove when removing eager_subs
```

Comment:
I've temporarily added a second test suite execution under the new subsitution changes in this PR for CI.  I will remove this as part of addressing #149.